api/controllers/ResumeController.ts

```diff
@@ -54,4 +57,11 @@ export class ResumeController {
     const resume = await this.resumeService.patchResume(params.uuid, patchResumeRequest.resume, user);
     return { error: null, resume: resume.getPublicResume() };
   }
+
+  @Get()
+  async getAllVisibleResumes(@AuthenticatedUser() user: UserModel): Promise<GetVisibleResumesResponse> {
+    if (!PermissionsService.canSeeAllVisibleResumes(user)) throw new ForbiddenError();
+    const resumes = (await this.resumeService.getVisibleResumes()).map((resume) => resume.getPublicResume());
+    return { error: null, resumes };
```

Comment:
nit: let's move the map operation to the return statement for easier readibility:
```suggestion
    const resumes = await this.resumeService.getVisibleResumes();
    return { error: null, resumes: resumes.map((resume) => resume.getPublicResume()) };
```